packages/react-components/src/components/SendBox.tsx

```diff
@@ -182,6 +243,18 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
     setTextValue(newValue);
   };
 
+  const renderErrorMessage = (fileUploads: FileUploadState[]): JSX.Element => {
+    let errorMsg: JSX.Element = <Stack></Stack>;
+    fileUploads &&
```

Comment:
unnecessary check since fileUploads is a required parameter

Reply:
This can be simplified
```tsx
const latestError = fileUploads.filter((fileUpload) => fileUpload.errorMessage).pop()
if (latestError) {
  return <Stack className={mergeStyles(errorBarStyle(theme))}>{latestError .errorMessage}</Stack>;
}
```